### ISSUE TITLE
Query not found

### ISSUE THREAD

**Issue by jenshavelberg** [mikaelkrief/VSTS-Extension-ScrumReport#8]
Hi mikael,
is the language a problem? We use a german tfs. 
See the following error:

> Error the query is not found :
> Create the query 'Scrum Report' on the root on Shared Queries folder

![image](https://cloud.githubusercontent.com/assets/6579164/14239708/e2f13d46-fa3e-11e5-9ddc-8fe344e41ff3.png)

Can you help me?

**Comment by mikaelkrief** [comment]
Hi Jess,
yes, maybe the problem is the language, because the extension search the query into "Shared query" and in germany the name is different.
Could see the console log in chrome or ie developper tools and send me the display error.

I 'll investigate for work with language.

thanks

**Comment by jenshavelberg** [comment]
of cource. thank you for your great support!

![image](https://cloud.githubusercontent.com/assets/6579164/14241267/e801a5b2-fa4a-11e5-9960-505d3272ecec.png)

**Comment by mikaelkrief** [comment]
thanks,
Can you send me the Url of the query page ?

**Comment by jenshavelberg** [comment]
did you mean that? 

wrong (en): https://tfs.vfm.de/tfs/vfmProjekte/vfm/_apis/wit/queries/Shared%20Queries%2FScrum%20Report

right (de):
https://tfs.vfm.de/tfs/vfmProjekte/vfm/_apis/wit/queries/Freigegebene%20Abfragen%2FScrum%20Report

**Comment by mikaelkrief** [comment]
with this 2 it does not work ?

**Comment by jenshavelberg** [comment]
no, in case of request the second url, the tfs returns status 200. all good

**Comment by mikaelkrief** [comment]
in wich case you call the first url ? your tfs language is in de ?

**Comment by jenshavelberg** [comment]
right. german tfs on premise

**Comment by jenshavelberg** [comment]
you see the screenshot (^^) i can't rename the shared querys folder... the foldername is still german.

**Comment by mikaelkrief** [comment]
for understand , for use the first url , you change the preference language ?

**Comment by jenshavelberg** [comment]
if i change the locale user profile language to english, the folder is still named in german. it's because we have used an german tfs installation. it works in no case.

**Comment by mikaelkrief** [comment]
Ok, Jess, I'll investigate for that extension work with all languages.
Thanks you for yours feedbacks, do not hesitate...

**Comment by twarnick** [comment]
Hi, we have also no 'Scrum Report' Query, but the url is correct.
Can you tell me where can i found the query text, so i can creating it?
